terraform/modules/ops/variables.tf

```diff
@@ -32,3 +32,8 @@ variable "setup_cd_system" {
   default     = false
   description = "Create deployment triggers. Enable only if Cloud Build has been granted GitHub access."
 }
+
+variable "e2e_api_url" {
```

Comment:
Adding a variable to the ops module may cause setup.sh to fail since we don't provide a value for it.

Reply:
Noted - that was intentional, but I made it optional for now based on our internal conversation.